scripts/build-probe-binaries

```diff
@@ -589,6 +609,7 @@ elif [ "OL6-UEK" = "$KERNEL_TYPE" ]; then
 	do
 		rhel_build $URL
 	done
+	error_logging
```

Comment:
because we need to account also for the build errors when this script runs in a container, I think that it is better to keep the errors (one entry per line) in a file shared between the host and the containers and read only one time all the errors from the file at the end of the script (line 600). If you invoke error_logging here and there are errors the script will end before compiling OL7.